src/ziggurat/config.clj

```diff
@@ -108,3 +110,51 @@
 (defn -get [^String key]
   (let [config-vals (get config (keyword key))]
     (java-response config-vals)))
+
+;; for backward compatibility
+(def ^:private mapping-table
+  {:auto-offset-reset-config           :auto-offset-reset
+   :changelog-topic-replication-factor :replication-factor
+   :consumer-group-id                  :group-id
+   :default-api-timeout-ms-config      :default-api-timeout-ms
+   :key-deserializer-class-config      :key-deserializer
+   :key-serializer-class               :key-serializer
+   :session-timeout-ms-config          :session-timeout-ms
+   :stream-threads-count               :num-stream-threads
+   :value-deserializer-class-config    :value-deserializer
+   :value-serializer-class             :value-serializer})
+
+(def ^:private non-kafka-config-keys
+  [:consumer-type
+   :producer
+   :channels
+   :oldest-processed-message-in-s
+   :origin-topic
+   :input-topics
+   :join-cfg])
```

Comment:
:poll-timeout-ms-config, :thread-count  from consumer namespace should be part of non-kafka-config-keys

Reply:
done